### ISSUE TITLE
Create Sequence Classification UDF

### ISSUE THREAD

**Issue by umitbuyuksahin** [exasol/transformers-extension#9]
Create an UDF for sequence text classification (please see [sequence-classification](https://huggingface.co/docs/transformers/task_summary#sequence-classification)) 
- takes variadic  inputs.
- returns softmax logits
- ex: transformers_sequence_classification(string, ..., string) -> softmax_logits
